e2e-tests/specs/relay-e2e.spec.ts

```diff
@@ -3,7 +3,7 @@ import { checkAuthState } from '../e2eTestUtils/helpers';
 
 test.describe.configure({ mode: 'parallel' });
 test.skip(({ browserName }) => browserName !== 'firefox', 'firefox only e2e!');
-test.describe('Relay e2e function email forwarding', () => {
+test.fixme('Relay e2e function email forwarding', () => {
```

Comment:
(Blocking) Is there a reason why we added fixme? Are we not able to fix this test?

Reply:
the available implementations for this are flaky, --- will re-add once we get a proper api for this, rather not run test if its flaky